api/swagger.yml

```diff
@@ -3156,6 +3156,11 @@ paths:
           schema:
             type: string
             format: date-time
+        - in: query
+          name: stop_at
+          description: Commit ID to stop at
```

Comment:
Is this a commit-ID or a ref?  There is no point exposing it to the user if we require  a commit ID.  AFAIK we _never_ require a commit ID as an API input, so I suggest using a ref here, too.

Reply:
From GetCommit API:
```
  /repositories/{repository}/commits/{commitId}:
    parameters:
      - in: path
        name: repository
        required: true
        schema:
          type: string
      - in: path
        name: commitId
        required: true
        schema:
          type: string
```

Or do you say it's a different case as it's part of the path and not a query input?

Reply:
This is different. Thr code literally takes a ref. And the test uses a ref.
We *must* call it "ref", and this must happen before the next release. Otherwise we break server.